xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-ui/src/main/resources/Main/DatabaseSearch.xml

```diff
@@ -101,7 +101,7 @@
 {{velocity}}
 #set ($rssMode = $xcontext.action == 'get' || "$!request.xpage" == 'plain')
 #if (!$rssMode)
-  #set ($discard = $xwiki.ssfx.use('uicomponents/search/search.css', true))
+  #set ($discard = $xwiki.linkx.use($services.webjars.url('org.xwiki.platform:xwiki-platform-search-webjar', 'search.min.css', {'evaluate': true}), {'type': 'text/css', 'rel': 'stylesheet'}))
```

Comment:
Line too long (more than 120 characters). The code could be formatted on multiple lines.

Reply:
I tend to ignore line length on templates and documents, since most of them already have extremely long lines and I'm pretty sure the checker ignores them. But yeah, I should probably fix it for readability.